tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -682,7 +683,7 @@ MINIJ-PAS PDR201912 Catalogue (December, 2019) is based on scientific images in
 
 -----
 
-**ChiVO TAP** https://vo.chivo.cl/tap ChiVO TAP service
+**ChiVO TAP** https vo.chivo.cl/tap ChiVO TAP service
```

Comment:
why this?

Reply:
The shed_lint command check that every url returns a 200 and unfortunately those are having some problems right now, so I thought it was the best way to keep the information and still pass the checks. It can be reversed once the archive have fix their issues

Reply:
ok, could you make a comment about that in the code so we know in the future?